Content.Server/Atmos/ExcitedGroup.cs

```diff
@@ -30,6 +30,12 @@ public void AddTile(TileAtmosphere tile)
             ResetCooldowns();
         }
 
+        public bool RemoveTile(TileAtmosphere tile)
+        {
+            tile.ExcitedGroup = null;
+            return _tile.Remove(tile);
```

Comment:
nitpick: why isn't `_tile` plural? it is a collection after all.

Reply:
good question, let me change that

Reply:
done